controllers/autovirtualdc_controller_test.go

```diff
@@ -232,6 +218,22 @@ var _ = Describe("AutoVirtualDC controller", func() {
 				err := k8sClient.Get(ctx, client.ObjectKey{Name: "test-avdc", Namespace: testNamespace}, avdc)
 				return apierrors.IsNotFound(err)
 			}).Should(BeTrue())
+
+			By("deleting VirtualDC")
+			if testcase.input.conditions != nil {
+				vdc := &nyamberv1beta1.VirtualDC{}
+				Eventually(func() error {
+					err := k8sClient.Get(ctx, client.ObjectKey{Name: "test-avdc", Namespace: testNamespace}, vdc)
+					return err
+				}).ShouldNot(HaveOccurred())
+				err = k8sClient.Delete(ctx, vdc)
+				Expect(err).NotTo(HaveOccurred())
+
+				Eventually(func() bool {
+					err := k8sClient.Get(ctx, client.ObjectKey{Name: "test-avdc", Namespace: testNamespace}, vdc)
+					return apierrors.IsNotFound(err)
+				}).Should(BeTrue())
+			}
```

Comment:
Is it necessary to delete VirtualDC resources here?
All VirtualDC resources are deleted in AfterEach, so I don't think it is necessary to delete them for cleaning purposes.

Reply:
I want to delete after each testcase is tested, but these tests is not in 'It'.
So I wrote procedure for delete VirtualDC.

Reply:
Sorry, I understand.